README.md

```diff
@@ -64,7 +64,8 @@ then go to `http://localhost:8080`.
 ## Usage
 
 - Clone this repo.
-- Run with `cargo run -- test.js` where `test.js` is an existing JS file with any JS valid code.
+- Run `cargo build`
+- Run with `cargo run -- test.js` where `test.js` is an existing JS file with any JS valid code. The file has to be in the root directory.
```

Comment:
It gives the person a clear idea to where the file should be added; it's worth mentioning.

Reply:
> It gives the person a clear idea to where the file should be added; it's worth mentioning.

It doesn't have to be in the root dir, it just require a path to the file. so `cargo run -- ../some-folder-outside-root/works.js`

Reply:
Although, alternative suggestions are always welcome

Reply:
> @HalidOdat , you are correct. But if somebody changes the directory to the target folder, and then runs `cargo-run`, then it will fail. I think, for better clarity, we should be mentioning it.

This, should be obvious, `cargo` doesn't work if `Cargo.toml` is not found (applies to pretty much every build system)

> Although, alternative suggestions are always welcome

We can make it explicit:

"Run with `cargo run -- test.js` in the project root directory where `test.js` is a path to an existing JS file with any JS valid code."

Reply:
Also, let's mention, if `cargo run` fails the first time, then run  `cargo build`.

Reply:
> Also, let's mention, if `cargo run` fails the first time, then run `cargo build`.

https://github.com/boa-dev/boa/pull/3092#discussion_r1249588880

Reply:
@HalidOdat I would request you to put suggestions in this PR. That would be helpful.

Reply:
It may be a bit more succinct to write.

```markup
 - Run with `cargo run -- test.js` in the root directory where `test.js` is a path to any existing valid JS file.
```

Do we need to double reference JS file and valid JS code?

Reply:
Double reference is not needed.

Reply:
> It may be a bit more succinct to write.
> 
> ```
>  - Run with `cargo run -- test.js` in the root directory where `test.js` is a path to any existing valid JS file.
> ```
> 
> Do we need to double reference JS file and valid JS code?

It looks perfect.